src/lib/y2storage/package_handler.rb

```diff
@@ -63,6 +63,7 @@ def reset
     # @return [Array<String>] new package list (may contain duplicates)
     def add_packages(pkg_list)
       @pkg_list.concat(pkg_list)
+      compact
```

Comment:
np: I would call this method `compact!` instead.

Reply:
I'm not so sure. In theory methods with an exclamation mark are only intended as a "more aggressive" counterpart of another method without the mark. There is no such counterpart here.